code/game/machinery/_machines_base/stock_parts/network_receiver.dm

```diff
@@ -6,7 +6,7 @@
 	icon_state = "net_lock"
 	part_flags = PART_FLAG_QDEL
 
-/obj/item/stock_parts/network_receiver/Initialize(ml, material_key)
+/obj/item/stock_parts/network_receiver/Initialize()
```

Comment:
If you remove those the compiler won't let you refer to them as named parameters with subclasses. And the auto-complete won't list them as parameters. It really not a good idea to remove them imo.

Reply:
Alright then.